Assets/LeapMotionModules/Attachments/Scripts/HandAttachments.cs

```diff
@@ -145,20 +149,15 @@ public class HandAttachments : IHandModel {
 
     public override bool SupportsEditorPersistence() { return true; }
 
-    private void OnDrawGizmos() {
-      DrawDebugLines();
-    }
-
     /** The colors used for each bone. */
     protected Color[] colors = { Color.gray, Color.yellow, Color.cyan, Color.magenta };
 
     /**
     * Draws lines from elbow to wrist, wrist to palm, and normal to the palm.
     * Also draws the orthogonal basis vectors for the pinch and grab points.
     */
-    protected void DrawDebugLines() {
-      RuntimeGizmoDrawer gizmoDrawer;
-      if (RuntimeGizmoManager.TryGetGizmoDrawer(gameObject, out gizmoDrawer)) {
+    public void OnDrawRuntimeGizmos(RuntimeGizmoDrawer gizmoDrawer) {
+      if (DrawHand && RuntimeGizmoManager.TryGetGizmoDrawer(gameObject, out gizmoDrawer)) {
```

Comment:
You don't need to try getting the gizmo drawer from inside the callback.  The callback is only called if a drawer is available.